_posts/2022/09/2022-09-05-eo-codestyle.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+date: 2022-09-05
+title: "EO codestyle"
+author: MCJOHN974
+---
+
+
+One of the goals of EOlang was clear and readable code. 
+That's why it has some codestyle checks processed by the compiler.
+
+Firstly, the level of nesting is controlled by the indentations.
+Reason – code with different indents is hard to read and understand,
+and code with correct indents is clear without parentheses. The
+size of indent is strict – two spaces. It was done for big decoration 
+hierarchy. Also, EOlang strictly controls all space symbols – no more
+than one whitespace between words, no more than one blank line between 
+objects and attributes, no blank lines in the code of one attribute,
+no tabs. If a code looks ugly without extra blank lines, it is a good 
+marker that you should create a new object or attribute.
```

Comment:
@MCJOHN974 Maybe, it might be better to split the text into points for better readability?
e.g.
```
1) the level of nesting...
2) The size of indent...
...
```
WDYT?

Reply:
I thought about that, but I find that points will be different size and it look not smart. But if you disagree I will add points

Reply:
@Graur

Reply:
@MCJOHN974 No, it is up to you